geode-management/src/main/java/org/apache/geode/management/api/ClusterManagementResult.java

```diff
@@ -47,6 +47,10 @@
     // configuration.
     // the operation is still deemed unsuccessful.
     FAIL_TO_PERSIST,
+    // async operation launched successfully
+    ACCEPTED,
```

Comment:
Why do we have both ACCEPTED and IN_PROGRESS? I see ACCEPTED set once we have submitted the async op.
But IN_PROGRESS is only set if we call checkStatus. Would it be better to get rid of ACCEPTED and set IN_PROGRESS only when ACCEPTED is currently set?

Reply:
they need to map to different HTTP STATUSes